Modules/Projects/Controllers/ProjectsController.php

```
@@ -128,10 +168,32 @@ public function view(int $projectId): \Illuminate\View\View
+        // Check if project exists
+        $project = $this->projectService->find($id);
+        if (!$project) {
+            return $this->redirectWithError('projects.index', TranslationHelper::trans('project_not_found'));
+        }
+
+        // Business rule: Cannot delete projects with related tasks
+        if (!$this->projectService->canDelete($id)) {
+            $blockers = $this->projectService->getDeletionBlockers($id);
+            $message = TranslationHelper::trans('project_deletion_not_allowed', [
+                'tasks' => $blockers['tasks'],
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Core\Support\TranslationHelper' in method 'delete'. (undefined)

(StaticAccess)